app/services/salesman/update.rb

```diff
@@ -0,0 +1,38 @@
+module Services
+  module Salesman
+    class Update
+      def initialize(id, name, status)
+        @name = name
+        @status = status
+        @id = id
+      end
```

Comment:
here you split object like this:

@name = update_params[:name]
@status = update_params[:status]
@id.      =. update_params[:id]